packages/clay-core/src/table/Cell.tsx

```diff
@@ -154,6 +179,14 @@ export const Column = React.forwardRef<HTMLTableCellElement, Props>(
 					<span className="text-truncate-inline">
 						<span className="text-truncate">{children}</span>
 					</span>
+				) : treegrid && index === 0 ? (
+					<div
+						style={{
+							paddingLeft: (level - 1) * 28,
+						}}
+					>
+						{children}
+					</div>
```

Comment:
I will also be adding autofit to allow you to add Checkbox and other elements you want in the cell.

```jsx
<Row items={columns}>
	{(column) => (
		<Cell key={`${row.id}:${column.id}`}>
			<Checkbox />
			<Icon symbol="folder" />
			{row[column.id]}
		</Cell>
	)}
</Row>
```

Reply:
Was the plan for this to be inline style, or is there some other utility class from @pat270 for this?

Reply:
Under the hood I'm using the `autofit` classes, the difference is that we don't make this visible to the dev so that he doesn't worry about implementing it.